common/src/main/java/com/datastrato/gravitino/dto/requests/TableUpdateRequest.java

```diff
@@ -157,9 +157,11 @@ public UpdateTableCommentRequest() {
      */
     @Override
     public void validate() throws IllegalArgumentException {
+      /*
```

Comment:
Spark supports remove comment though unset table properties `comment`, @jerryshao @mchades  do you think is it reasonable to make `newComment` field in `UpdateTableCommentRequest` null to represent remove comment?